lua/entities/kleiner/cl_init.lua

```diff
@@ -0,0 +1,142 @@
+-- This file is subject to copyright - contact swampservers@gmail.com for more information.
+-- INSTALL: CINEMA
+
+if(SERVER)then return end
+include("shared.lua")
+language.Add("kleiner", "Dr. Isaac Kleiner")
+
+function ENT:Initialize()
+		local effectdata = EffectData()
+	effectdata:SetOrigin( self:GetPos() )
+	effectdata:SetEntity( self )
+	util.Effect( "propspawn", effectdata )
+end
+ 
+
+local spkr = Material("voice/icntlk_pl")
+hook.Add("CreateClientsideRagdoll","KleinerRagdollsFix",function(ent,rag)
```

Comment:
Does the ragdoll color not work otherwise?

Reply:
SetPlayerColor only exists on player entity, so the material proxy typically only returns the blue color unless you add that function to an entity.